apps/minaintyg/src/components/FilterAccordion/FilterAccordion.tsx

```diff
@@ -0,0 +1,17 @@
+import { IDSIconChevron } from '@frontend/ids-react-ts'
+import { ReactNode } from 'react'
+
+export function FilterAccordion({ title, children }: { title: string; children: ReactNode }) {
```

Comment:
Can be made to a general component. Only the text "Filtrera lista" is unique for Filter.